app/src/renderer/vuex/modules/send.js

```diff
@@ -74,7 +74,7 @@ export default ({ node }) => {
         gas: args.gas,
         gas_prices: args.gas_prices,
         simulate: args.simulate,
-        memo: `Sent via Lunie`
+        memo: `${args.memo ? args.memo + ` ` : ``}(Sent via Lunie)`
```

Comment:
imho this goes against what the user inputs on the memo input field. You should either prefill the input field with `Sent via Lunie` as a default. You don't know if the transaction that's required on exchanges will be valid for them if you add the extra `Sent via Lunie`.